src/tools/compiletest/src/runtest.rs

```diff
@@ -359,6 +361,27 @@ impl<'test> TestCx<'test> {
         self.check_forbid_output(&output_to_check, &proc_res);
     }
 
+    fn run_crash_test(&self) {
+        let pm = self.pass_mode();
+        let proc_res = self.compile_test(WillExecute::No, self.should_emit_metadata(pm));
+
+        if std::env::var("COMPILETEST_VERBOSE_CRASHES").is_ok() {
+            eprintln!("{}", proc_res.status);
+            eprintln!("{}", proc_res.stdout);
+            eprintln!("{}", proc_res.stderr);
+            eprintln!("{}", proc_res.cmdline);
+        }
+
+        // if a test does not crash, consider it an error
+        if proc_res.status.success() || matches!(proc_res.status.code(), Some(1 | 0)) {
+            self.fatal(&format!(
+                "test no longer crashes/triggers ICE! Please give it a mearningful name, \
```

Comment:
don't mind me, I'm just passing by

```suggestion
                "test no longer crashes/triggers ICE! Please give it a meaningful name, \
```

Reply:
I think oli already fixed this in a drive-by cleanup somewhere 😅

Reply:
Ah, did he? I just saw that in one of this PRs https://github.com/rust-lang/rust/pull/126316#issuecomment-2162663178

Reply:
I remember doing that, but it probably hasn't landed yet